packages/app/features/auth/sign-up/sign-up-form.tsx

```diff
@@ -80,166 +95,208 @@ export const SignUpForm = () => {
+              $gtMd={{
+                mt: '0',
+                als: 'flex-end',
+                mx: 0,
+                ml: 'auto',
+                w: '$10',
+                h: '$3.5',
+              }}
+            >
+              <ButtonText size={'$2'} padding={'unset'} ta="center" margin={'unset'} col="black">
+                {'/SIGN UP'}
+              </ButtonText>
+            </SubmitButton>
+          </XStack>
+          <YStack pos="relative" pb={'$size.1'} w="100%" maw="$size.22">
+            <XStack jc="center" ai="center" mt="$4" gap="$1">
+              <Paragraph size="$2" color="$color11">
+                Already have an account?
+              </Paragraph>
+              <SubmitButton onPress={() => handleSignIn()} disabled={isSigningIn} unstyled>
+                <ButtonText color="$color11" size="$2" textDecorationLine="underline">
+                  {isSigningIn ? 'Signing in...' : 'Sign in'}
+                </ButtonText>
+              </SubmitButton>
+            </XStack>
+          </YStack>
+          <YStack pt="$4">
+            {!!process.env.NEXT_PUBLIC_TURNSTILE_SITE_KEY && (
+              <Turnstile
+                siteKey={process.env.NEXT_PUBLIC_TURNSTILE_SITE_KEY}
+                onSuccess={(token) => {
+                  setCaptchaToken(token)
+                }}
+              />
+            )}
+          </YStack>
+        </YStack>
+      )}
+    >
+      {({ countrycode: CountryCode, phone: Phone }) => (
+        <YStack gap="$5" jc="center" $sm={{ f: 1 }}>
+          <BigHeading color="$color12">CREATE YOUR ACCOUNT</BigHeading>
+          <H3
+            lineHeight={28}
+            $platform-web={{ fontFamily: '$mono' }}
+            $theme-light={{ col: '$gray10Light' }}
+            $theme-dark={{ col: '$olive' }}
+            fontWeight={'300'}
+            $sm={{ size: '$5' }}
+          >
+            Sign up with your phone number.
+          </H3>
+
+          <YStack gap="$4">
+            <Paragraph color="$color12" size={'$1'} fontWeight={'500'}>
+              Your Phone
+            </Paragraph>
+            <XStack gap="$5">
+              {CountryCode}
+              {Phone}
+            </XStack>
+          </YStack>
+        </YStack>
+      )}
+    </SchemaForm>
+  )
+
+  const backUpForm = (
+    <YStack gap="$5" jc="center" $sm={{ f: 1 }} maw={535}>
+      <BigHeading color="$color12">ARE YOU BACKING UP?</BigHeading>
+      <H3
+        lineHeight={28}
+        $platform-web={{ fontFamily: '$mono' }}
+        $theme-light={{ col: '$gray10Light' }}
+        $theme-dark={{ col: '$olive' }}
+        fontWeight={'300'}
+        $sm={{ size: '$5' }}
+      >
+        This number is already associated with an account.
+      </H3>
+      <XStack gap="$4" mt="$7">
+        <SubmitButton size="$4" w="$12" onPress={handleBackUpConfirm}>
+          <ButtonText>YES</ButtonText>
+        </SubmitButton>
+        <Button
+          borderColor="$primary"
```

Comment:
for light mode primary is always assigned to black. Unfortunately our tamagui doesn't do this for us yet


```suggestion
          borderColor="$primary"
          $theme-light={{borderColor: "$color12"}}
```